_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Visualizing alerts and anomalies using OpenSearch Dashboards applications"
+authors: 
+  - jdbright
+  - ashisagr
+  - ohltyler
+  - amgalitz
+  - ashwinpc
+  - hnyng
+  - joshuarrrr
+  - xeniatup
+date: 2023-07-20
+meta_keywords: alerting, anomaly detection, alert, anomaly, observabilty, dashboard, dashboards, visualizations, visualize
+meta_description: Integrating alerts and anomalies with OpenSearch Dashboard visualizations.
```

Comment:
Updated meta: 
**meta_keywords:** anomaly detection, OpenSearch observability, OpenSearch dashboards, OpenSearch Alerting Plugin
**meta_description:** Learn how you can create anomaly detectors and alerting monitors directly from your OpenSearch Dashboard line chart visualizations with select aggregations.

Reply:
Looks like the meta description provided was assigned to the Twitter Card. Please update the meta description with what I provided.